worker/hybrid/cluster_api.md

```diff
@@ -0,0 +1,82 @@
+Cluster API for IronWorker
```

Comment:
@derekschultz can you put 1-2 sentences in here with high level context of what this API is for